src/integration/pages/frontPage/user.canonical.js

```diff
@@ -0,0 +1,13 @@
+const { canonical } = global;
+
+export default () => {
+  it('I can see atleast a section', () => {
+    const sect = canonical.document.querySelector('section');
```

Comment:
A better query to use would be [`getByRole`](https://testing-library.com/docs/dom-testing-library/api-queries#byrole)

```suggestion
  it('I can see the top stories section', () => {
    const topStoriesEl = canonical.getByRole('region');
```

You can see that the sections have a role attribute by checking the HTML:

![Screenshot 2020-04-06 at 08 28 01](https://user-images.githubusercontent.com/4798332/78533299-a605d080-77e0-11ea-866c-9ba8b12b5c0f.png)

Reply:
I dont want to test here specifically top stories ...wanted to assert whether the frontpage has atleast one section which can be topstories,audio-video ..etc